perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -31,12 +31,19 @@ olcAttributeTypes: {26}( 1.3.6.1.4.1.8057.2.80.30 NAME 'loa' DESC 'Level of auth
 olcAttributeTypes: {27}( 1.3.6.1.4.1.8057.2.80.31 NAME 'internalUserIdentifiers' DESC 'All identifiers collected from UserExtSource attribute additionalIdentifiers' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {28}( 1.3.6.1.4.1.8057.2.80.32 NAME 'perunFacilityDn' DESC 'DN of Facility the resource belongs to' EQUALITY distinguishedNameMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.12 SINGLE-VALUE )
 olcAttributeTypes: {29}( 1.3.6.1.4.1.8057.2.80.35 NAME 'eduPersonOrcid' DESC 'All OrcID identifiers of user' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {30}( 1.3.6.1.4.1.8057.2.80.36 NAME 'checkGroupMembership' DESC 'OIDC check group membership' EQUALITY booleanMatch SYNTAX  1.3.6.1.4.1.1466.115.121.1.7 )
```

Comment:
Boolean syntax for attribute seems to be problematic, since from perun/java will provide "true", "false", but ldap syntax expect "TRUE" and "FALSE". We will need to provide own ``ValueTransformer`` for such attributes (java+context usage) or handle it more generally in ``SingleAttributeValueExtractor`` for all boolean atributes.
Also using plain string values in LDAP is possible, but it might be more desirable to use boolean. I will discuss it internally and let you know.

Reply:
Ok, I realized, that in Perun, boolean attributes are either "true" or null, since we do not store false values. False can be probably returned only from virtual attributes, but in audit messages no event "set to false" is logged. It is converted to null and stored as "removed".

So please change all boolean attributes to string syntax in ldap with single-value. Proxy will check such attributes on presence. Thank you.

Reply:
Changed syntax and matching rules to string for relevant attributes.